low_voltage/mqtt.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
this file must be unittested. 
i understand hmp.py cannot, but you can mock it.